src/Framework/Migrations/MigrationsServiceProvider.php

```diff
@@ -30,6 +30,8 @@ public function boot()
     {
         Hooks::addAction('admin_init', ManualMigration::class, '__invoke', 0);
         Hooks::addAction('admin_init', MigrationsRunner::class, 'run', 0);
-        Hooks::addAction('give_upgrades', MigrationsRunner::class, 'run', 0);
+        //Hooks::addAction('give_upgrades', MigrationsRunner::class, 'run', 0);
+        // running batch actions via cron doesn't trigger give_upgrades and all registered actions fail
```

Comment:
Will this break the existing migrations that are using the old upgrade system?

Reply:
I believe not, but to be safe, I left the `give_upgrades` action and changed `admin_init` to `action_scheduler_init`.